types/material-ui/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for material-ui v0.16.0
+// Type definitions for material-ui v0.16.1
```

Comment:
I can't find this added property in 0.16.1 (http://www.material-ui.com/v0.16.1/#/components/flat-button) but I find it in 0.17.2...is this intended?